pkgs/development/python-modules/chia-rs/default.nix

```diff
@@ -0,0 +1,49 @@
+{ buildPythonPackage
+, lib
+, fetchFromGitHub
+, pytestCheckHook
+, rustPlatform
+}:
+
+buildPythonPackage rec {
+  pname = "chia_rs";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "chia-network";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-4TIRj7FMIArI/EvDARReC4MqDG44zjn/MKoUHAVqq5s=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+  };
+
+  postPatch = ''
+    cp ${./Cargo.lock} Cargo.lock
+  '';
+
+  nativeBuildInputs = with rustPlatform; [
+    cargoSetupHook
+    maturinBuildHook
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  buildAndTestSubdir = "wheel";
+
+  preBuild = ''
+    # avoid ENOENT in maturinBuildHook
+    touch wheel/Cargo.lock
+  '';
+
+  meta = with lib; {
+    description = "Rust crate & wheel with consensus code";
+    homepage = "https://www.chia.net/";
```

Comment:
Could we link to something specific to this package instead of the super generic homepage?

Reply:
Not really. It's first-party and very generic. There's the [GH page](https://github.com/Chia-Network/chia_rs/), but all it says is:

> This cargo workspace contains code useful for working with the Chia network.
>
> It contains a rust crate chia (here), a python wheel api chia_rs (in wheel), and web assembly (in wasm).

Reply:
That would be better especially from a reviewer perspective.

Reply:
Fixed